book/chapters/chapter3/evaluation_and_benchmarking.qmd

```diff
@@ -874,50 +852,43 @@ These visualizations are also available for `r ref("ResampleResult")` objects.
 In this case, the predictions of individual resampling iterations are merged before calculating a ROC or PR curve (micro averaged):
 
 ```{r performance-058}
-#| output: false
-#| cache: false
+#| eval: false
```

Comment:
should this be eval: false?

Reply:
yes, the code is actually evaluated in the next chunk